### PR TITLE
fix: unauthorized error message

### PR DESCRIPTION
**Pull request by sanjay-k1910**
## What?
Updated unauthorized error message when a user tries to use API with the invalid access token
Previous response:
```
{
  "statusCode": 500,
  "message": "Cannot read properties of null (reading 'iss')",
  "error": "Internal Server error"
}
```
Refactored response:
```
{
  "statusCode": 401,
  "message": "Invalid access token",
  "error": "Unauthorized"
}
```

## Why?
If an unauthorized user tries to access API then it should return a proper error message.

### CODE REVIEW

File: libs/common/src/response-messages/index.ts [hunk 1]
```diff
@@ -59,7 +59,8 @@ export const ResponseMessages = {
             invitationStatusUpdateInvalid: 'Status update is invalid. Request is already',
             resetSamePassword: 'New password should not be the current password',
             resetPasswordLink: 'Unable to create reset password token',
-            invalidResetLink: 'Invalid reset password link or expired'
+            invalidResetLink: 'Invalid reset password link or expired',
```
Comment (ankita-p17) [inline libs/common/src/response-messages/index.ts:62]: update verbose
"Invalid or expired reset password link"

File: libs/common/src/response-messages/index.ts [hunk 2]
```diff
@@ -59,7 +59,8 @@ export const ResponseMessages = {
...
+            invalidAccessToken: 'Invalid access token'
```
Comment (ankita-p17) [inline libs/common/src/response-messages/index.ts:63]: 'Unauthorized' or 'Authentication failed' message is also fine. No need to provide technical details to user.

File: libs/common/src/response-messages/index.ts (hunk 1 above)
Comment (sanjay-k1910) [reply libs/common/src/response-messages/index.ts]: Resolved.

Comment (sanjay-k1910) [reply libs/common/src/response-messages/index.ts]: Resolved.

File: libs/common/src/response-messages/index.ts (hunk 2 above)
Comment (sanjay-k1910) [reply libs/common/src/response-messages/index.ts]: Resolved.

**Review by ankita-p17 — APPROVED** [review]

**Pull request merged**